index.d.ts

```diff
@@ -12,13 +13,20 @@ export interface QRCodeProps {
   color?: string;
   /* the color of the background */
   backgroundColor?: string;
+  /* SVG to render as logo.
+  * Can be either a svg string or a React component if you're using ex: '@svgr/webpack' or similar
+  * In case both this prop `logo` is provided, then this takes precedence and `logo` will be ignored. */
+  logoSVG?: React.FC<SvgProps> | string;
   /* an image source object. example {uri: 'base64string'} or {require('pathToImage')} */
   logo?: ImageSourcePropType;
   /* logo size in pixels */
   logoSize?: number;
   /* the logo gets a filled rectangular background with this color. Use 'transparent'
          if your logo already has its own backdrop. Default = same as backgroundColor */
   logoBackgroundColor?: string;
+  /* If the logo is provided via SVG this color will be set as it's `fill` property
+  * it does nothing if logo is provided as an Image source object */
```

Comment:
```suggestion
  /* If the logo is provided via `logoSVG` prop, this color will be set as it's `fill` property,
  * otherwise it does nothing */
```